packages/node-cli/src/CLI.js

```diff
@@ -37,9 +39,12 @@ module.exports = class CLI {
 
 	start(process) {
 		try {
-			const runInInteractiveMode = this._isRunningInInteractiveMode();
-
 			const commandMetadataList = this._commandsMetadataService.getCommandsMetadata();
+			this._validateCommandExists(commandMetadataList, process.argv[2]);
```

Comment:
process.argv[2]);    will never be out of index? for example i call 'suitecloud'. isn't the args just 2? 0 and 1.

Reply:
No, it would be "undefined" and then in the if of line 73 would not enter.

Reply:
great.